tests/cases/elements/menubar_for.slint

```diff
@@ -25,7 +25,10 @@ export component TestCase inherits Window {
                 for num in 45 : MenuItem {
                     title: "Recent " + num;
                     activated => {
-                        result += "Recent " + num;
+                        if self.title != "Recent " + num {
+                            result += "ERROR: invalid self.title";
```

Comment:
```suggestion
                            result += "Error: invalid self.title";
```

Oops, the tree-sitter test fails because the output contained `ERROR` as it matched this string